api/ServiceWorkerContainer.json

```diff
@@ -622,7 +622,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerContainer/startMessages",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
This should be `true`. 

modules/service_worker/service_worker_container.idl

Reply:
It's from https://chromium.googlesource.com/chromium/src/+/1eef889053f1b30d62acccc34b1a3e66c5afd5e8 and shipping in M74. I'll revert this do a separate PR.

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/3696